roles/shiftstack/README.md

```diff
@@ -0,0 +1,10 @@
+# shiftstack
+Please explain the role purpose.
```

Comment:
Something like: 

Role for triggering openshift on openstack QA automation

Reply:
Done